terraform/src/common/container_app_environment.tf

```diff
@@ -1,5 +1,5 @@
 module "container_app_env" {
-  source = "git::https://github.com/nhsuk/dct.terraform-modules.container-app-env?ref=1.0.2"
+  source = "git::https://github.com/nhsuk/dct.terraform-modules.container-app-env?ref=059c2f3e5f2c556c9a19388ffc646d1af3b11546"
```

Comment:
Update to tag after https://github.com/nhsuk/dct.terraform-modules.container-app-env/pull/3 is merged and tagged